tests/actions/init.test.ts

```diff
@@ -1,255 +1,124 @@
   });
 
   afterEach(() => {
-    jest.restoreAllMocks();
+    vi.restoreAllMocks();
   });
 
   test("if both requirements are missing, then the execution fails", async () => {
-    // Given
-    simServCheckInstallRequirements.mockResolvedValue({git: false, docker: false});
+    simServCheckInstallRequirements.mockResolvedValue({ git: false, docker: false });
 
-    // When
     await initAction(defaultActionOptions, simulatorService);
 
-    // Then
-    expect(error).toHaveBeenCalledTimes(1);
     expect(error).toHaveBeenCalledWith(
-      "Git and Docker are not installed. Please install them and try again.\n",
+      "Git and Docker are not installed. Please install them and try again.\n"
     );
   });
 
   test("if only docker is missing, then the execution fails", async () => {
-    // Given
-    simServCheckInstallRequirements.mockResolvedValue({git: true, docker: false});
+    simServCheckInstallRequirements.mockResolvedValue({ git: true, docker: false });
 
-    // When
     await initAction(defaultActionOptions, simulatorService);
 
-    // Then
-    expect(error).toHaveBeenCalledTimes(1);
     expect(error).toHaveBeenCalledWith("Docker is not installed. Please install Docker and try again.\n");
   });
 
   test("if only git is missing, then the execution fails", async () => {
-    // Given
-    simServCheckInstallRequirements.mockResolvedValue({git: false, docker: true});
+    simServCheckInstallRequirements.mockResolvedValue({ git: false, docker: true });
 
-    // When
     await initAction(defaultActionOptions, simulatorService);
 
-    // Then
-    expect(error).toHaveBeenCalledTimes(1);
     expect(error).toHaveBeenCalledWith("Git is not installed. Please install Git and try again.\n");
   });
 
   test("if check install requirements fail, then the execution aborts", async () => {
-    // Given
     simServCheckInstallRequirements.mockRejectedValue(new Error("Error"));
 
-    // When
     await initAction(defaultActionOptions, simulatorService);
 
-    // Then
-    expect(error).toHaveBeenCalledTimes(1);
     expect(error).toHaveBeenCalledWith(new Error("Error"));
   });
 
   test("if both versions are too low, then the execution fails", async () => {
     const mockVersionNumber = "99.9.9";
+    simServCheckVersionRequirements.mockResolvedValue({
+      node: mockVersionNumber,
+      docker: mockVersionNumber,
+    });
 
-    // Given
-    simServCheckVersionRequirements.mockResolvedValue({node: mockVersionNumber, docker: mockVersionNumber});
-
-    // When
-    await initAction(defaultActionOptions, simulatorService);
-
-    // Then
-    expect(error).toHaveBeenCalledTimes(1);
-    expect(error).toHaveBeenCalledWith(
-      `Docker version ${mockVersionNumber} or higher is required. Please update Docker and try again.\nNode version ${mockVersionNumber} or higher is required. Please update Node and try again.\n`,
-    );
-  });
-
-  test("if only docker version is too low, then the execution fails", async () => {
```

Comment:
@epsjunior why do you want to remove these ones?

Reply:
@cristiam86  the if statements were already covered by other tests, but you're right, it's better to create specific tests for these cases to make the coverage clearer and more targeted.

Evidences:
https://github.com/yeagerai/genlayer-cli/pull/115/commits/fa6337d15fc1afed0dbc28fd37250907f90e26ca
<img width="578" alt="Screenshot 2024-11-06 at 06 17 56" src="https://github.com/user-attachments/assets/ce104810-254e-4c6a-8b7a-50c3def97aec">